sklearn/metrics/ranking.py

```diff
@@ -40,7 +40,9 @@ def auc(x, y, reorder=False):
     """Compute Area Under the Curve (AUC) using the trapezoidal rule
 
     This is a general function, given points on a curve.  For computing the
-    area under the ROC-curve, see :func:`roc_auc_score`.
+    area under the ROC-curve, see :func:`roc_auc_score`.  For an alternative
+    way to summarize a precision-recall curve, see
```

Comment:
I don't understand. AUC is area under the ROC curve, not PR curve.

Reply:
@amueller Per `This is a general function, given points on a curve` and the Davis and Goadrich paper, my understanding was that AUC can refer to trapezoidal area under any curve.  In the Davis and Goadrich terminology we can have AUC-ROC or AUC-PR.

What change do you recommend?

Reply:
never mind, I think your version is fine.

Reply:
I'm updating the `roc_auc_score`  description to say it computes ROC AUC (the acronym used in `_binary_roc_auc_score`) instead of the more general AUC.